### DISCUSSION TITLE
Bag

### DISCUSSION THREAD

**Discussion by Hamidulla-python** [aclap-dev/video-downloadhelper#652] [Help and Q&A]
### Your question

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://maqsadonline.uz/student/courses/e0cedfbc-78bb-4088-bffc-8f24220a3ce9/lessons/ce7a149d-d68d-4cdb-b349-5261cfd93ca0",
  "error": {
    "details": "SideDownload error: : [dash @ 00000226e53cb6c0] Error when loading first fragment of playlist\r\n[in#0 @ 00000226e53cb540] Error opening input: Invalid data found when processing input\r\nError opening input file https://kinescope.io/f314de33-f65c-4b43-b5e9-bbf71b7a8809/master.mpd?expires=1716659224&kinescope_project_id=44aaf4c6-aa1b-4a44-b502-c2fa06affc03&sign=d90260c2a60ee54a15561d635fccce51.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2546385949068650",
    "error": "coapp_failure",
    "id": "coapp_failure_2bafb758-4b48-498a-9225-8f206e01fe32"
  }
}

### Addon & CoApp version and details

a

**Comment by Mattmjs** [comment]
It looks like this is DRM protected, ffmpeg only says _error when loading first fragment of playlist._
Another tool I use says this:

[generic] Extracting URL: https://kinescope.io/f314de33-f65c-4b43-b5e9-bbf71b7a8809/master.mpd?expires=1716659224&kinescope...e54a15561d635fccce51
[generic] master: Downloading webpage
WARNING: [generic] Falling back on generic information extractor
[generic] master: Extracting information
ERROR: [generic] master: **This video is DRM protected**
